### DISCUSSION TITLE
Recent article about challenges of generating WebAssembly

### DISCUSSION THREAD

**Discussion by konsoletyper** [konsoletyper/teavm#782] [Announcements]
Recently I wrote an [article](https://habr.com/ru/articles/757182/) on Habr about challenges of producing WebAssembly from JVM bytecode. Habr is quite popular blog platform for Russian-speaking developers. Unfortunately, the article is in Russian, but if you don't speak Russian, you can try to use ChatGPT to translate it. If you speak Russian and want to translate the article into English and publish somewhere, you are welcome, don't only forget to refer to original article and mention me as an author.

**Comment by rmcdouga** [comment]
Google Translate did a pretty good job.  I was able to comprehend most of the article. 

Here's a link: https://habr-com.translate.goog/ru/articles/757182/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp&_x_tr_hist=true 

Thanks for writing and sharing this!
